File: 2148-count-elements-with-strictly-smaller-and-greater-elements/2148-count-elements-with-strictly-smaller-and-greater-elements.cpp
class Solution {
public:
    int countElements(vector<int>& nums) {
        sort(nums.begin(), nums.end());
        int n, count;
        n=nums.size();
        count=0;
        for(int i=1; i<n-1;) {
            if(nums[i-1]<nums[i]) {
                int j=i+1;
                while(j<n && nums[j]==nums[i])
                    j++;
                if(j<n) {
                    count+=j-i;
                    i=j-1;
                }
            }
            i++;
        }
        return count;
    }
};

File: 2148-count-elements-with-strictly-smaller-and-greater-elements/2148-count-elements-with-strictly-smaller-and-greater-elements.py
class Solution:
    def countElements(self, nums: List[int]) -> int:
        maxNum = max(nums)
        minNum = min(nums)
        if maxNum == minNum:
            return 0
        n = len(nums)
        return n - (nums.count(maxNum) + nums.count(minNum))